pkg/localize/locales/en/cmd/kafka_update.en.toml

```diff
@@ -36,30 +39,49 @@ one = 'Name of the Kafka instance you want to update'
 description = 'Description for the --owner flag'
 one = 'ID of the user you want to set as the owner of this Kafka instance'
 
+[kafka.update.flag.reauthentication]
+one = 'Enable connection reauthentication for the Kafka instance'
+
 [kafka.update.flag.yes]
 one = 'Forcibly update the Kafka instance without confirmation'
 
 [kafka.update.summaryTitle]
 one = 'Summary of Proposed Changes'
 
+[kafka.update.reauthentication.disclaimer]
+one = '''
+If a bad actor obtains credentials or service account to Kafka instance that has disabled reauthentication,
+deactivating the user account or service account will not disconnect already established connections.
+
+Options to disconnect client when reauthentication is disabled:
+- Contact Red Hat support
+- Use ACLs to prevent unauthorized connections from performing any operation
+'''
+
 [kafka.update.confirmDialog.message]
 one = 'Are you sure you want to update the Kafka instance "{{.Name}}"?'
 
-[kafka.update.log.info.sameOwnerNoChanges]
-one = 'Nothing to update: user "{{.Owner}}" is already the owner of the Kafka instance "{{.InstanceName}}".'
-
 [kafka.update.log.info.updating]
 one = 'Updating Kafka instance "{{.Name}}"...'
 
+[kafka.update.log.info.nothingToUpdate]
+one = 'Nothing to update'
+
 [kafka.update.log.info.updateSuccess]
 one = 'Kafka instance "{{.Name}}" has been updated. Run "rhoas kafka describe --name {{.Name}}" to view its configuration.'
 
+[kafka.update.log.info.updateFailed]
+one = 'Kafka instance could not be updated.'
+
 [kafka.update.log.info.loadingUsers]
 one = 'Loading users...'
 
 [kafka.update.input.message.selectOwner]
 one = "Select new owner:"
 
+[kafka.update.input.message.reauthentication]
+one = "Enable connection reauthentication [optional]:"
```

Comment:
```suggestion
one = "Enable connection reauthentication"
```

Reply:
As every flag is optional unless validated

Reply:
added the [optional] tag in line with `kafka topic update` interactive. Basically to imply that it could be left blank

Reply:
Ah.. this is for interactive mode. Sorry I missed that mode. Please ignore.